.github/workflows/pull-request-checks.yaml

```diff
@@ -382,8 +382,8 @@ jobs:
       - name: Run tests
         run: cd build; ctest . -V -L THOROUGH -j2
 
-  check-macos-10_15-make-clang:
-    runs-on: macos-10.15
+  check-macos-11-make-clang:
+    runs-on: macos-11
```

Comment:
Should we then perhaps get rid of `check-macos-11-cmake-clang` and be content that we have a `make`-based test for macOS 11 and a `CMake`-based test for macOS 12?

Reply:
I diffed the two jobs, and they are essentially the same:

```diff
fotiskoutoulakis@192 ~ % diff /tmp/12.yml /tmp/11.yml 
1,2c1,2
< check-macos-12-cmake-clang:
<     runs-on: macos-12
---
> check-macos-11-cmake-clang:
>     runs-on: macos-11
```

So, the only difference I would reckon is the system toolchain (and other environment parameters), which if they are appropriately exercised by the `cmake` build, then I don't see any reason to keep it.

Reply:
Actually, I went ahead and did just that.